dev/src/dev/search.clj

```diff
@@ -0,0 +1,61 @@
+(ns dev.search
+  (:require
+   [metabase.search.postgres.core :as search.postgres]
+   [metabase.search.postgres.core-test :refer [legacy-results]]
```

Comment:
```suggestion
   [metabase.search.postgres.index-test :refer [legacy-results]]
```
I think you moved it

Reply:
Thanks for spotting. Fixed in the next PR so won't both updating this one - plan to squash them anyway.